webroot/img/flags/nap.svg

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   version="1.0"
+   width="600"
+   height="400"
+   id="svg698"
```

Comment:
You will have to remove this `id="svg698"` because it messes up with the build of the language icons asset.

I updated the wiki article about adding a new language and mentioned this issue in [step 2](https://github.com/Tatoeba/tatoeba2/wiki/Adding-a-new-language#2-add-the-language-icon):

> - Open the file with a text editor:
>   - Remove comments if there are any.
>   - Remove the "id" attribute from the <svg> tag if there is any.
>   - Not doing so will trigger an error with the asset build (see for example [issue #2911](https://github.com/Tatoeba/tatoeba2/issues/2911)).